src/pages/PlanPage/subRoutes/CancelPlanPage/CancelPlanPage.tsx

```diff
@@ -37,11 +56,16 @@ function CancelPlanPage() {
 
   const discountNotApplied =
     !accountDetailsData?.subscriptionDetail?.customer?.discount
-  const showDiscount =
+  const showSpecialOffer =
     discountNotApplied && isMonthlyPlan(accountDetailsData?.plan?.value)
+  const showTeamSpecialOffer =
+    multipleTiers &&
+    shouldDisplayTeamCard({ plans }) &&
+    !isTeamPlan(accountDetailsData?.plan?.value)
```

Comment:
Would this work if we had enterprise/basic/gh marketplace (`users`) plans here?

I wonder if we could do something instead of `!isTeamPlan` to `isProPlan` and do something like this


``` javascript
export function isProPlan(plan) {
  if (isString(plan)) {
    return !isUndefined(isSentryPlan(plan)) || !isUndefined(isCodecovProPlan(plan))
  }
  return false
}

```
 (and we'd also need to make the isCodecovProPlan method)

Reply:
so plan should be showing for:
users-pr-inappm
users-pr-inappy
users-sentrym
users-sentryy

any other plans?

Reply:
For those yup!